src/lib/validationSchemas/specialistCreateSchema.js

```diff
@@ -0,0 +1,57 @@
+import { z } from 'zod';
+import { FormatOfWork, Gender } from '@prisma/client';
+import { PHONE_REGEX } from '@/lib/consts';
+
+const zString = fieldName => z
+  .string({
+    required_error: `${fieldName} is required`,
+    invalid_type_error: 'Must be a string',
+  })
+  .trim();
+
+const zStringWithMinMax = fieldName => zString(fieldName)
+  .min(2, {
+    message: 'Can not be empty',
+  })
+  .max(128, {
+    message: 'Must not be longer than 128 characters',
+  });
+
+const zArray = z.string().array().min(1);
```

Comment:
🟠 thats `zStringArray` , not just any array :)

Reply:
NOTE: Issue found. Please **DO NOT** merge.